sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/CompactibleFileStreamLog.scala

```diff
@@ -140,7 +137,9 @@ abstract class CompactibleFileStreamLog[T <: AnyRef : ClassTag](
     out.write(Serialization.write(entry).getBytes(UTF_8))
   }
 
-  private def deserializeEntry(line: String): T = Serialization.read[T](line)
+  private def deserializeEntry(line: String): T = {
```

Comment:
why need change this line?

Reply:
reverted, This is test how to use 'Manifest [T]', but I forgot how to reverse it.